### ISSUE TITLE
Register GTalkSMS for the share Intent (android.intent.action.SEND)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#242]
```
It would be nice if user could send URL's back to notify gtalk address directly 
from browser.

This would remove need for Phone 2 Chrome or similar applications and browser 
extensions.

More advanced options might send pictures, but I would be very happy with just 
urls ;-)
```

Original issue reported on code.google.com by `dpavlin` on 5 Jan 2012 at 10:24

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately that is not so easy. We would need C2DM or an XMPP instance 
running within the browser.
```

Original comment by `fschm...@gmail.com` on 23 Jan 2012 at 3:55
- Changed title: **Send URLs to notificaton Address from browser**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I'm don't think that browser receiver for C2DM/XMPP is necessary. Intent 
listener in gtalksms could just send url back to management account,
```

Original comment by `dpavlin` on 24 Jan 2012 at 10:20

**Comment by GoogleCodeExporter** [comment]
```
Oh so you are talking about an Android browser. We already register GTalkSMS to 
handle http(s) URIs on Android in the manifest. Which means if you are 
presented with an "open URL with" dialog on Android, GTalkSMS will also show 
up, besides the installed browsers. But I don't think that there is a way to 
send an URL from the browser to GTalkSMS (besides via clipboard), because a 
browser will always open a URL itself.
```

Original comment by `fschm...@gmail.com` on 24 Jan 2012 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
I'm sorry for confusion, I'm a newbie in Android terminology. Let me try to 
describe use-case better:

I am surfing in Android browser. I stumbled upon page which can't be viewed 
correctly on it, so I would love to open it in my desktop browser. I select 
menu > Share Page and I get various application registered for 
android.intent.action.SEND intent. I would love to get URL from that intent in 
my gtalksms management account.

I think you are speaking about android.intent.action.VIEW which opens URLs on 
Android device.
```

Original comment by `dpavlin` on 24 Jan 2012 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
Now I get it. :) Good idea!
```

Original comment by `fschm...@gmail.com` on 25 Jan 2012 at 8:21
- Changed title: **Register GTalkSMS for the share Intent (android.intent.action.SEND)**
- Changed state: **Accepted**
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 20 Feb 2012 at 10:58
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8f7b4978d74f.
```

Original comment by `fschm...@gmail.com` on 20 Feb 2012 at 11:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.5
```

Original comment by `Florent....@gmail.com` on 2 Apr 2012 at 9:38
- Changed state: **Deployed**
